library/camel-kamelets-catalog/src/main/resources/kamelets/jira-add-comment-sink.kamelet.yaml

```diff
@@ -0,0 +1,74 @@
+    camel.apache.org/provider: "Red Hat"
+    camel.apache.org/kamelet.group: "JIRA"
+  labels:
+    camel.apache.org/kamelet.type: "sink"
+spec:
+  definition:
+    title: "Jira Add Comment Sink"
+    description: |-
+      Add a new comment to an existing issue in Jira.
+
+      The Kamelet expects the following headers to be set:
+
+      - `issueKey` / `ce-issueKey`: as the issue code.
+
+      The comment is set in the body of the message.
+
+    required:
+      - jiraUrl
+      - username
+      - password
+    type: object
+    properties:
+      jiraUrl:
+        title: Jira URL
+        description: The URL of your instance of Jira
+        type: string
+        example: http://my_jira.com:8081
+      username:
+        title: Username
+        description: The username to access Jira
+        type: string
+        x-descriptors:
+        - urn:camel:group:credentials
+      password:
+        title: Password
+        description: The password or the API Token to access Jira
+        type: string
+        format: password
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:password
+        - urn:camel:group:credentials
+  types:
+    in:
+      mediaType: application/json
+  dependencies:
+  - "camel:core"
+  - "camel:jackson"
+  - "camel:jira"
+  - "camel:kamelet"
+  - "mvn:com.fasterxml.jackson.datatype:jackson-datatype-joda:2.12.5"
```

Comment:
Is there any productized one for this?

Reply:
Yes, [there is 2.12.4](https://mvnrepository.com/artifact/com.fasterxml.jackson.datatype/jackson-datatype-joda?repo=redhat-ga) should we use it ?